cdap-docs/admin-manual/source/operations/monitoring.rst

```diff
@@ -15,3 +15,6 @@ See the :ref:`Logging <http-restful-api-logging>`, :ref:`Metrics <http-restful-a
 and :ref:`Monitoring <http-restful-api-monitor>` HTTP APIs, and the
 :ref:`Java Client <reference:java-client-api>` for more information.
 
+CDAP can be monitored using external systems such as `Nagios <https://www.nagios.org/>`__; a Nagios-style plugin 
```

Comment:
Do you think this page warrants sub-headers?  looking at the rendered view i thought it might help... as this last sentence can be easily overlooked.  Up to you though.